docs/transactions.md

```diff
@@ -834,10 +834,10 @@ Gets all child certificates for the given certificate.
 Revoked certificates are not returned.
 
 - Parameters:
-  - subject: `string`  - certificates's `Subject`
+  - subject: `string(Base64)`  - certificates's `Subject`
   - subject_key_id: `string`  - certificates's `Subject Key Id`
```

Comment:
Add description also for `subject_key_id`:
  - subject_key_id: `string`  - certificates's `Subject Key Id` in hex string format